mw-webapp/src/logic/wayPage/labels/LabelLine.module.scss

```diff
@@ -2,21 +2,40 @@
 
 .labelLine {
   align-items: baseline;
-  gap: $gapBig;
+  gap: 0 $gapBig;
+
+  @media only screen and (max-width: $mediaWidthTablet) {
+    flex-wrap: wrap;
+    padding-bottom: $paddingExtraMedium;
+  }
 }
 
 .labelContainer {
   display: flex;
-  min-width: 120px;
-  max-width: 150px;
+  min-width: 250px;
+  max-width: 250px;
```

Comment:
is it okey that malelMaxWidth is 270? but labelContainer is max 250? How container could be less then element?